### ISSUE TITLE
Deleted accounts won't be deleted

### ISSUE THREAD

**Issue by ben-utzer** [friendica/friendica#8780]
### Expected behavior
If an account in /admin/users is deleted it should get deleted soon after (currently usually a few days). 

### Actual behavior
The counter of days left till deletion turns "negative", saying "2 days ago" for the column "Permanent deletion". 

### Steps to reproduce the problem
Create an account, then delete it and wait. 

### Friendica version you encountered the problem
current RC 2020.06

### Friendica source (git, zip)
git

### PHP version
PHP 7.4.x

### SQL version
mariadb Ver 15.1

**Comment by AlfredSK** [comment]
But the deletion works. Shortly after the 'two days ago' is displayed the user vanishes completely from the list.

**Comment by ben-utzer** [comment]
OK, so just the countdown and the waittime don't fit together? I will wait another 2 days, it now says "3 days ago" and the deletion was (more than) a week ago.

**Comment by AlfredSK** [comment]
Yep. It begins with "in 5 days" or so. Then it switched to "X days ago". And then it goes away.

**Comment by ben-utzer** [comment]
now "4 days ago" the deletion will take place. I still think this is some bug.

**Comment by ben-utzer** [comment]
At some point they were gone. So the bug is the number of days shown is wrong. Deletion take place.
